frontend/components/app.js

```diff
@@ -46,48 +27,75 @@ class App extends React.Component {
   }
 
   render() {
-    const { children } = this.props;
-    const storeState = this.state;
+    const { user, children, notifications } = this.props;
 
-    if (storeState.user.isLoading) {
+    if (user.isLoading) {
```

Comment:
Safe to assume that `user` is never undefined? I can't seem to make it break so I think so.